core/src/test/java/org/elasticsearch/recovery/RecoveryWhileUnderLoadIT.java

```diff
@@ -255,19 +256,21 @@ public void testRecoverWhileRelocating() throws Exception {
             logger.info("--> refreshing the index");
             refreshAndAssert();
             logger.info("--> verifying indexed content");
-            iterateAssertCount(numShards, indexer.totalIndexedDocs(), 10);
+            iterateAssertCount(numShards, (int) indexer.totalIndexedDocs(), 10);
```

Comment:
I'm missing the point of this casts; the corresponding method parameter is a `long`?